RELEASES.md

```diff
@@ -1,3 +1,144 @@
+Version 1.85.0 (2025-02-20)
+==========================
+
+<a id="1.85.0-Language"></a>
+
+Language
+--------
+- [Support var-args for more calling conventions](https://github.com/rust-lang/rust/pull/116161) 
+  This stabilizes usage of `...` variable args in extern declarations for `system`, `aapcs`, `sysv64`, `win64`, and `efiapi` calling conventions. `C` and `cdecl` were previously stable.
```

Comment:
This was reverted in #136897, and pending beta backport in #136980.
(I'll close the relnotes issue too.)